test/simplifyExpression/basicsSearch/basicsSearch.test.js

```diff
@@ -28,6 +28,9 @@ describe('simplify basics', function () {
     ['2x*2*-1', '2x * 2 * -1'], // does not remove multiplication by -1
     // removeExponentByOne
     ['x^1', 'x'],
+    // removeExponentBaseOne
+    ['1^3', '1'],
```

Comment:
can you add another test for if the exponent isn't a constant yet, but will eventually be? that's actually the best case for the use of this rule I think!

e.g. `1^(2 + 3 + 5/4 + 7 - 6/7)` would be simplified to 1 right away without dealing with those messy fractions :)